internal/util/function/common.go

```diff
@@ -0,0 +1,82 @@
+/*
+ * # Licensed to the LF AI & Data foundation under one
+ * # or more contributor license agreements. See the NOTICE file
+ * # distributed with this work for additional information
+ * # regarding copyright ownership. The ASF licenses this file
+ * # to you under the Apache License, Version 2.0 (the
+ * # "License"); you may not use this file except in compliance
+ * # with the License. You may obtain a copy of the License at
+ * #
+ * #     http://www.apache.org/licenses/LICENSE-2.0
+ * #
+ * # Unless required by applicable law or agreed to in writing, software
+ * # distributed under the License is distributed on an "AS IS" BASIS,
+ * # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * # See the License for the specific language governing permissions and
+ * # limitations under the License.
+ */
+
+package function
+
+const (
+	InsertMode string = "Insert"
+	SearchMode string = "Search"
+)
+
+// common params
+const (
+	modelNameParamKey    string = "model_name"
+	dimParamKey          string = "dim"
+	embeddingUrlParamKey string = "url"
+	apiKeyParamKey       string = "api_key"
+)
+
+// ali text embedding
```

Comment:
if those variables are per provider, consider moving it to their own package? so that those are also under correct namespace: `ali.TextEmbeddingV1`, `openai.TextEmbeddingAda002`.

also for env variable names, add EnvName suffix